### DISCUSSION TITLE
[issue forms] Incorrectly saying there's an issue with my template

### DISCUSSION THREAD

**Discussion by AnonymousWP** [community/community#110091] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I believe there's a bug related to GitHub's issue forms:

As you can see in this repo, GitHub says there's an issue with the template: https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/blob/281-featissue-templates-use-issue-forms/.github/ISSUE_TEMPLATE/3-add-new-packages.yml

![image](https://github.com/community/community/assets/50231698/a7d6c860-15ee-42b7-aea4-a25ed137a9b6)

First of all, the link is not even clickable (the "learn more about this error" one), and second, there's no other template with that same name. Just take a look at the repo (and its branch). There's no other file in that branch/PR that has the same value of `name`.

When I pushed this to a private repo, this error did not occur. In fact, everything (except required checkboxes of course, as that's not included for private repos) worked completely fine.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by MarcelloFernsAtVMFarms** [comment]
Hello.
I just had the same issue.
In my case, I had two .md files with unique names but the same `name:` within each file.
If I use your screenshot as an example,
I had two files `general issue template.md` and `unique issue template.md` that both had `name: Add new packages(s)` within the file. Changing the `name:` to be unique solved the issue in my case.

Hope this helps. Cheers.

**Reply by AnonymousWP** [reply]
Hi, thanks for your reply. I remember I already checked that back at the time, because I indeed know the names must be unique.
